api/envoy/config/route/v3/route.proto

```diff
@@ -134,6 +134,12 @@ message RouteConfiguration {
   // NOTE: this option will not strip the port number (if any) contained in route config
   // :ref:`envoy_v3_api_msg_config.route.v3.VirtualHost`.domains field.
   bool ignore_port_in_host_matching = 14;
+
+  // Ignore path-parameters in path-matching.
```

Comment:
Should we add new field to RouteMatch to enable the user match the path parameters? But that should be separated PR and requirement.

Reply:
Sorry, probably I didn't describe it clearly. I mean a new field for people matching the `path parameters`, since we have a field to match the path, and a field to match the query parameter. There is no field for `path parameters`, you only can match that with path currently, but I think that won't work with multiple path parameters with different orders.

Reply:
ahh, I agree that it should be in another PR. 
I am not sure path-parameters is a widely used legacy feature, (same as this PR, not many users are using it), but would be nice if Envoy can support it.